pixeltable/utils/pytorch.py

```diff
@@ -70,15 +70,23 @@ def _unmarshall(self, k: str, v: Any) -> Any:
         if self.column_types[k].is_image_type():
             assert isinstance(v, bytes)
             im = PIL.Image.open(io.BytesIO(v))
+            arr = np.array(im)
             if self.image_format == "np":
-                return np.array(im)
-            else:
-                assert self.image_format == "pt"
-                import torchvision  # pylint: disable = import-outside-toplevel
-                return torchvision.transforms.ToTensor()(im)
+                return arr
+
+            assert self.image_format == "pt"
+            import torchvision  # pylint: disable = import-outside-toplevel
+            return torchvision.transforms.ToTensor()(arr)
```

Comment:
leave comment explaining why you're not passing in the pil image (which according to the docs is legal).

Reply:
why is this necessary?